### PR TITLE
[PLAT-937] Adds methods for secure compute endpoints

### PR DESCRIPTION
**Pull request by lily-tsou**
- Adds methods for secure compute endpoints
- Adds tests for new methods

### CODE REVIEW

File: types/computation.js
Comment (lily-tsou) [inline types/computation.js:1]: Forgot to write a comment here. Will do that before merging!

File: types/computation.js
Comment (lily-tsou) [inline types/computation.js:14]: And will add newline :)

**Comment by Mrhea** [comment]
And my sdk methods are coming soon!

File: lib/storage/client.js
Comment (rosasck) [inline lib/storage/client.js:1576]: feel free to push this to master to unblock matthew but can you add documentation to the read me!

**Review by Mrhea — APPROVED** [review]

**Pull request merged**
